Makefile

```diff
@@ -1,18 +1,23 @@
-IMAGE_NAME=secrets-store-csi-driver-provider-azure
+-include secrets.env
+export $(shell test -f secrets.env && sed 's/=.*//' secrets.env)
+
 REGISTRY_NAME ?= upstreamk8sci
 REGISTRY ?= $(REGISTRY_NAME).azurecr.io
 DOCKER_IMAGE ?= $(REGISTRY)/public/k8s/csi/secrets-store/provider-azure
 IMAGE_VERSION ?= 0.0.5
+PROJECT_NAME ?= secrets-store-csi-driver-provider-azure
```

Comment:
Can we keep `IMAGE_NAME` instead of `PROJECT_NAME`? `PROJECT_NAME` was used only for `REPO_PATH` and we can keep that the same?